seed.js

```diff
@@ -98,6 +98,12 @@ const costumes = [
     discount: discounts[Math.floor(Math.random() * (discounts.length - 1))],
     madeIn: faker.address.country(),
   },
+  {
+    costumeName: 'Tarzan',
+    price: 33.43,
+    categortyId: 2,
+    imageUrl: '/costumeImages/disney/tarzan.jpeg',
+  },
```

Comment:
can you add the discount, material, madeIn seeding (you can copy paste from another costume)

Reply:
updated!